frontend/.storybook/mockData.ts

```diff
@@ -0,0 +1,85 @@
+import { auditLogFactory } from "../src/fixtures/auditLog";
+import { destructionListAssigneesFactory } from "../src/fixtures/destructionList";
+import { FIXTURE_SELECTIELIJSTKLASSE_CHOICES } from "../src/fixtures/selectieLijstKlasseChoices";
+import { userFactory, usersFactory } from "../src/fixtures/user";
+import { zaaktypeChoicesFactory } from "../src/fixtures/zaaktypeChoices";
+
+export const MOCKS = {
```

Comment:
Should these replace the ones in the `frontend/.storybook/preview.ts`?

Reply:
Ah you've also added them, I'm happy to merge this setup importing them in `preview.ts`.

- [ ] Merge mock setup.